src/python/pants/backend/jvm/tasks/jvm_compile/execution_graph.py

```diff
@@ -300,14 +304,15 @@ def submit_jobs(job_keys):
 
       while not status_table.are_all_done():
         try:
-          finished_key, result_status, value = finished_queue.get(timeout=10)
+          finished_key, result_status, value, duration = finished_queue.get(timeout=10)
```

Comment:
Did you intend to assign `finished_queue.get(timeout=10)` to `duration` instead of `value`?

Reply:
yes, because the `finished_queue` now contains `result` which is a tuple that contains the `duration` 
see here - https://github.com/pantsbuild/pants/pull/8441/files#diff-cfd8b0aa6dbd9f3ac89ccbc4f3b10b32R288

Reply:
Ah 👍 Thanks! That was me being confused by the python syntax.